libcloudforensics/providers/gcp/internal/storage.py

```diff
@@ -173,3 +175,63 @@ def DeleteObject(self, gcs_path: str) -> None:
     gcs_objects = self.GcsApi().objects()
     request = gcs_objects.delete(bucket=bucket, object=object_path)
     request.execute()  # type: Dict[str, Any]
+
+  def GetBucketSize(self,
+                    bucket_name: str = None,
```

Comment:
If I recall correctly, optional arguments should be typed accordingly to match the rest of the codebase: `bucket_name: Optional[str] = None`